lms/djangoapps/verify_student/models.py

```diff
@@ -1046,7 +1051,9 @@ def add_status_from_checkpoints(cls, checkpoints, user, status):
             None
         """
         for checkpoint in checkpoints:
-            cls.objects.create(checkpoint=checkpoint, user=user, status=status)
+            # get 'location_id' from last entry and add it in new entry
+            location_id = cls.objects.filter(checkpoint=checkpoint)[0].location_id
```

Comment:
@awais786 @aamir-khan @wedaly I am assuming here that on `SoftwareSecurePhotoVerification` callback there will be always an entry in `VerificationStatus` related to given `checkpoint` from which we will get the `location_id` of `ReverificationBlock`.